lib/money_column/active_record_hooks.rb

```diff
@@ -5,11 +5,13 @@ def self.included(base)
     end
 
     module ClassMethods
```

Comment:
Maybe we should deprecate this entire class. AR provides built-in support for BigDecimal data types and with [composed_of](http://api.rubyonrails.org/v5.0.0/classes/ActiveRecord/Aggregations/ClassMethods.html) it can do the same thing money_column does, and more (using it in finder methods).